tls/_common/test/test_prf.py

```diff
@@ -157,3 +163,107 @@ def test_sha384_148bytes(self):
         )
         actual_output = prf(secret, label, seed, hashes.SHA384(), 148)
         assert actual_output == expected_output
+
+
+def ascii_bytes(min_size, max_size):
```

Comment:
It *may* be generally useful, but I'm not sure how much we'll end up using Hypothesis.  I'd like to leave it here for now.